bin/tat

```diff
@@ -2,7 +2,8 @@
 #
 # Attach or create tmux session named the same as current directory.
 
-session_name="$(basename "$PWD" | tr . -)"
+path_name="$(basename "$PWD" | tr . -)"
+session_name=${1-$path_name}
```

Comment:
Interesting! I assumed that `${x-"default"}` would print out the value of `$x`, then the literal string `-default`. It looks like this behaves exactly the same as `${x:-"default"}`, i.e. it falls back only if `$x` (or, in this case, `$1`), is not set.

I recommended double quotes because what if the path name has spaces? But it seems like it handles them sanely even without double quotes.